acra/configuring-maintaining/optimisations/obtaining_metrics.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Obtaining metrics
+weight: 9
+---
+
+# Obtaining metrics
+
+Acra services provide some metrics for Jaeger when configured to do so. This includes
+[AcraConnector]({{< ref "acra/configuring-maintaining/general-configuration/acra_connector/_index.md#jaeger" >}}),
+[AcraServer]({{< ref "acra/configuring-maintaining/general-configuration/acra_server/_index.md#jaeger" >}}),
+[AcraTranslator]({{< ref "acra/configuring-maintaining/general-configuration/acra_translator/_index.md#jaeger" >}}).
+
+# Metrics
+
+<!-- decryptor/postgresql/pg_decryptor.go and decryptor/mysql/response_proxy.go -->
+* `ProxyClientConnection` checks every client request using AcraCensor, if request is allowed, sends it to the database (PostgreSQL or MySQL)
+  * `ProxyClientConnectionLoop` corresponds to one exact query being checked and processed
```

Comment:
This trace represents one iteration of processing packet from client's db driver to database and ends on finishing processing. Driver establishes connection with AcraServer/DB and it may be idle some time until it sends new request serialized as several packets in db specific protocol. This trace tracks one db packet processing.